### ISSUE TITLE
Shift++ (combo with key '+')

### ISSUE THREAD

**Issue by bennyjonsson** [chieffancypants/angular-hotkeys#244]
Can't set key '+' as combo key
`hotkeys.add({
                combo: 'alt++',` // key +

**Comment by tomsoderlund** [comment]
Me too, can't get `'mod+plus'` or `'mod+shift+='` (as described in https://github.com/ccampbell/mousetrap/issues/243) to work either.

**Comment by eassa** [comment]
it workes for me using Ctrl ++ ,  (the + in the num lock panel only ) 
    hotkeys.add({
            combo: 'ctrl++',
            description: 'Zoom IN ',
            callback: function(event, hotkey) {
                event.preventDefault();
                $scope.zoomin();
            }
        });
        // zoom out 
        hotkeys.add({
            combo: 'ctrl+-',
            description: 'Zoom Out',
            callback: function(event, hotkey) {
                event.preventDefault();
                $scope.zoomout();
            }
        });

**Comment by tomsoderlund** [comment]
Yeah I only have a laptop keyboard and it doesn't work there. Seems to be differences between numeric keyboard and regular - saw this in the Mousetrap discussions too.
